cupy/fft/_callback.pyx

```diff
@@ -199,6 +199,7 @@ cdef inline void _mod_compile(str tempdir, str mod_name, str obj_host) except*:
 # other SMs always use sm_X0's symbols only and do not have their own
 # specializations.
 cdef dict _cc_major_map = {
+    '9': ('90',),
     '8': ('80', '86', '87'),
```

Comment:
```suggestion
    '8': ('80', '86', '87', '89'),
```

Reply:
Thanks for the suggestion. I was wondering about that... Simply adding '89' (compute capability for Ada Lovelace) would cause an error with CUDA 11.7 or older, right?

Reply:
Good point, you're right Akira-san. I'll take a closer look at CTK 11.8 later and check if there's anything we can extract for sm89 from `libcufft_static.a`.